schulcloud/FilesViewController.swift

```diff
@@ -119,32 +118,48 @@ class FilesViewController: UITableViewController, NSFetchedResultsControllerDele
     }
 
     override func tableView(_ tableView: UITableView, canEditRowAt indexPath: IndexPath) -> Bool {
-        guard let file = self.fetchedResultsController.sections?[indexPath.section].objects?[indexPath.row] as? File else { return false }
-        return file.permissions.contains(.write)
+        guard let currentUser = Globals.currentUser else { return false }
```

Comment:
It's currently possible to edit the three base folders and the course folders. Please add a guard statement for this.